src/components/Dialogs/StakingProvider.tsx

```diff
@@ -22,7 +23,13 @@ import {
 } from "react";
 import { StakingSource } from "./StakingDialog/StakingDialog";
 
-const supportedPools: StakingPool[] = [LINEAR_POOL, STNEAR_POOL];
+const getSupportedPools = (networkId: string): StakingPool[] => {
+  const base = [LINEAR_POOL, STNEAR_POOL];
+  if (networkId === "mainnet") {
+    return [...base, RNEAR_POOL];
+  }
```

Comment:
Nit: The `networkId` will always be `mainnet` now, so I don't think this check does anything in practice at the moment